qc_schema/dev/properties/scf_properties.py

```diff
@@ -0,0 +1,71 @@
+"""
+The complete list of SCF level properties.
+"""
+
+scf_properties = {}
+
+scf_properties["scf_one_electron_energy"] = {
+    "description": "The one-electron energy contribution [H] to the total SCF energy.",
```

Comment:
Here and later on, what is `[H]`?

Reply:
The core Hamiltonian, I dont really look at these descriptions as complete (they are not intended to be), but more about the general scaffold to build on.